internal/common/install.go

```diff
@@ -14,10 +13,6 @@ import (
 const (
 	Config                   = "configure"
 	Verify                   = "verify"
-	IamAssessment            = "iam-assessment"
```

Comment:
Moved these to internal/provider/resources/install/resources.go under the `installresources` package.